sdk/go/tests/client_test.go

```diff
@@ -31,16 +32,16 @@ func TestEncoding(t *testing.T) {
         t.Error("Private key is different after encoding/decoding")
     }
 
-    pubstr := MustEncode(GenPubKey(priv))
+    pubstr := hex.EncodeToString(GenPubKey(priv))
     if pubstr != PUBSTR {
         t.Error("Public key doesn't match expected. Got", pubstr)
     }
-    pub := MustDecode(pubstr)
+    pub, _ := hex.DecodeString(pubstr)
     if len(pub) != 33 {
         t.Error("Encoded pubkey wrong length. Should be 33, but is", len(pub))
     }
```

Comment:
Prior to your changes, this was testing the Must{En,De}code() functions. Since we don't need to test these anymore, L34-42 can be simplified to:
```
if hex.EncodeToString(GenPubKey(priv)) != PUBSTR {
    t.Error("Public key doesn't match expected. Got", pubstr)
}
```